### ISSUE TITLE
Can't customize printable content height (IOS)

### ISSUE THREAD

**Issue by matthewoj** [katzer/cordova-plugin-printer#181]
I have exhausted all options regarding printing a label (62mm x 2.4in) to a brother label printer (QL-810W).

The content sent to the printer is: 62mm x 1000mm for the following cases:
- Printing a single line of text: `print('<span style="height: 62mm; width: 2.4in">Sample</span>', OPTIONS)`
- Printing a DOM element: `print(document.getElementById('id'), OPTIONS)`

I have also attempted mutating the bounds variable, printing in landscape & portrait and the printer is still always thinking the content is 1000mm tall, therefore printing an excessive amount of white-space.

Furthermore, while attempting to interrogate this issue, I attached a background color to the content sent to be printed, thinking that the container of the content is being stretched to 1000mm? but this is not the case, the result of this case is a 62mm x 1000mm length of paper with a single 62mm x 2.4in rectangle containing the respective background-color, leaving an excessive amount of paper to waste (blank). I then thought maybe I need to explicitly set the margin=0 on the printable element? but this is not the case.

In summary, I need to be able to print a 62mm x 2.4in label to a printer rather than 62mm x 1000mm page containing the 62mm x 2.4in label.

**Comment by matthewoj** [comment]
I ended up using cordovas Photo Viewer plugin & utilise the share feature to print images.

**Comment by katzer** [comment]
The master branch already includes the 2 properties `paperHeight` and `paperWidth`.

**Comment by matthewoj** [comment]
Even with those parameters set in the options, it was not respected. Anyhow 1 year to respond to this is just another reason to avoid using this fault package

**Comment by newelement** [comment]
@matthewoj is correct the ngx params do not include the paperWidth and paperHeight.

`export interface PrintOptions {
    /**
     * The name of the print job and the document
     */
    name?: string;
    /**
     * The network URL of the printer.
     * Only supported on iOS.
     */
    printerId?: string;
    /**
     * Specifies the duplex mode to use for the print job.
     * Either double-sided (duplex:true) or single-sided (duplex:false).
     * Double-sided by default.
     * Only supported on iOS
     */
    duplex?: boolean;
    /**
     * The orientation of the printed content, portrait or landscape
     * Portrait by default.
     */
    landscape?: boolean;
    /**
     * If your application only prints black text, setting this property to true can result in better performance in many cases.
     * False by default.
     */
    grayscale?: boolean;
    /**
     * The Size and position of the print view
     */
    bounds?: number[] | any;
}`
